lib/archethic_web/controllers/faucet_controller.ex

```diff
@@ -78,47 +78,44 @@ defmodule ArchethicWeb.FaucetController do
     end
   end
 
-  defp transfer(
-         recipient_address,
-         curve \\ Crypto.default_curve()
-       )
-       when is_bitstring(recipient_address) do
-    {gen_pub, _} = Crypto.derive_keypair(@pool_seed, 0, curve)
+  @curve Crypto.default_curve()
```

Comment:
FYI, when you do this, you add a _compilation time dependency_  `Crypto -> FaucetController`, meaning everytime you compile `Crypto`, you will have to compile `FaucetController`. 

I suggest to avoid this, and directly use it in the function content so it is a _runtime dependency_.

Reply:
👍